frontend/package.json

```diff
@@ -34,12 +34,14 @@
     "react-syntax-highlighter": "^15.5.0",
     "tailwind-merge": "^2.3.0",
     "vite": "^5.3.3",
+    "vite-tsconfig-paths": "^4.3.2",
```

Comment:
Should be installed as a dev dependency (https://github.com/aleclarson/vite-tsconfig-paths?tab=readme-ov-file#usage)

Reply:
`ESLint: 'vite-tsconfig-paths' should be listed in the project's dependencies, not devDependencies.(import/no-extraneous-dependencies)`

it will throw a warning from ESLint if it is placed in the dev dep. is this warning correct? i'd like to hear your opinion and am happy to let it return to the dev dep.

Reply:
ESLint flags it because it treats it as a runtime dependency, when it is actually a development/build dependency (further confirmed by the official documentation). Vite manages the build process and takes care of resolving the imports properly, so `vite-tsconfig-paths` is only helpful during development.

You can safely ignore the warning with `/* eslint-disable import/no-extraneous-dependencies */`

Reply:
okay, let me adjust it